server/api-util/sdk.js

```diff
@@ -96,6 +96,16 @@ exports.handleError = (res, error) => {
         statusText: error.message,
       })
       .end();
+  } else if (error.message == 'Account has ongoing transactions with Stripe states') {
```

Comment:
Perhaps this could be formulated a bit differently. Perhaps something like this:

```js
  } else if (error.message == 'User has transactions on states that include incomplete payment processing') {
```

"Stripe states" sounds quite strange - and if we / customizers add other payment providers, it would be outdated.